common/djangoapps/enrollment/tests/fake_data_api.py

```diff
@@ -21,6 +21,8 @@
 
 _ENROLLMENT_ATTRIBUTES = []
 
+_VERIFIED_MODE_EXPIRED = []
```

Comment:
This shouldn't be a list. As the code is written, it's being used as a boolean. However, you presumably want to be able to set the verified mode's expiration status per-course, so it would make sense to make this a set of course IDs whose verified mode is expired.